src/workloads/execution/DensifyFillCombo.yml

```diff
@@ -0,0 +1,181 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of queries that combine $densify and $fill stages.
+
+GlobalDefaults:
+  dbname: &db test
+  batchSize: &batchSize 30000
+  fieldName: &field "numeric"
+  index: &index
+    keys: {numeric: 1}
+    options: {name: "numeric"}
+  smallStepSize: &smallStep {^RandomInt: {min: 3, max: 3}}
+  nop: &Nop {Nop: true}
+
+Actors:
+- Name: InsertData
```

Comment:
Should we do this with a timeseries collection?

Reply:
Would you like to? It wasn't my understanding that we needed to do this in a timeseries collection, but I can if you want?

Reply:
done